twitter_scraper/modules/profile.py

```diff
@@ -49,71 +50,111 @@ def __parse_profile(self, page):
-        if not self.location:
+        try:
+            self.location = html.find('div.location')[0].text
+            if not self.location:
+                self.location = None
+        except:
             self.location = None
 
-        self.birthday = html.find(".ProfileHeaderCard-birthdateText")[0].text
-        if self.birthday:
-            self.birthday = self.birthday.replace("Born ", "")
-        else:
+        # TODO cannot find ProfileHeaderCard-birthdateText
+        try:
+            self.birthday = html.find(".ProfileHeaderCard-birthdateText")[0].text
+            if self.birthday:
+                self.birthday = self.birthday.replace("Born ", "")
+            else:
+                self.birthday = None
+        except:
             self.birthday = None
 
-        self.profile_photo = html.find(".ProfileAvatar-image")[0].attrs["src"]
+        try:
+            self.profile_photo = html.find("td.avatar img")[0].attrs["src"]
+        except:
+            self.profile_photo = None
 
+        # TODO cannot find ProfileCanopy-headerBg
         try:
             self.banner_photo = html.find(".ProfileCanopy-headerBg img")[0].attrs["src"]
-        except KeyError:
+        except:
             self.banner_photo = None
 
-        page_title = html.find("title")[0].text
-        self.name = page_title[: page_title.find("(")].strip()
-
-        self.user_id = html.find(".ProfileNav")[0].attrs["data-user-id"]
+        try:
+            page_title = html.find("title")[0].text
+            self.name = page_title[: page_title.find("(")].strip()
+        except:
+            self.name = None
+        
+        try:
+            self.user_id = html.find(".ProfileNav")[0].attrs["data-user-id"]
+        except:
+            self.user_id = None
 
-        self.biography = html.find(".ProfileHeaderCard-bio")[0].text
-        if not self.birthday:
-            self.birthday = None
+        try:
+            self.biography = html.find("div.bio div.dir-ltr")[0].text     
+            if not self.biography:
+                self.biography = None
+        except:
+            self.biography = None
 
-        self.website = html.find(".ProfileHeaderCard-urlText")[0].text
-        if not self.website:
+        try:
+            self.website = html.find("div.url div.dir-ltr")[0].text
+            if not self.website:
+                self.website = None
+        except:
             self.website = None
 
+        # get stats table if available
+        stats_table = None
+        stats = None
+        try:
+            stats_table = html.find('table.profile-stats')[0]
+            stats = stats_table.find('td div.statnum')
+            if not stats:
+                self.tweets_count = None
+                self.following_count = None
+                self.followers_count = None
+        except:
+            self.tweets_count = None
+            self.following_count = None
+            self.followers_count = None
```

Comment:
```suggestion
        try:
            stats_table = html.find('table.profile-stats')[0]
            stats = stats_table.find('td div.statnum')
        except:
            stats_table = None
            stats = None
```
How about this?

Reply:
@aldnav  Thank you for your suggestion. Your suggestion is definitely better!
I just make an adjustment in this refactor
https://github.com/bisguzar/twitter-scraper/pull/167/commits/f13ec7ff78cea5cf146a535a8d653d4d38f7d4fa